src/ios/CDVSplashScreen.m

```diff
@@ -187,7 +187,35 @@ - (NSString*)getImageName:(UIInterfaceOrientation)currentOrientation delegate:(i
     {
         imageName = @"Default";
     }
-    
+
+    // Add Asset Catalog specific prefixes
+    if ([imageName isEqual:@"LaunchImage"])
+    {
+        if (UI_USER_INTERFACE_IDIOM() == UIUserInterfaceIdiomPhone)
```

Comment:
you have the device, so you can do a if (device.iPhone), and you have device.iPhone5, device.iPhone6, device.iPhone6Plus and device.iPad available too, so you can use them instead of checking the size again